src/main/java/com/raoulvdberge/refinedstorage/integration/overloaded/IntegrationOverloaded.java

```diff
@@ -0,0 +1,11 @@
+package com.raoulvdberge.refinedstorage.integration.overloaded;
+
+import net.minecraftforge.fml.common.Loader;
+
+public class IntegrationOverloaded {
+
+    public static boolean isLoaded() {
+        return Loader.isModLoaded("overloaded");
```

Comment:
Could be nice to return a cached value which is initialized in a static block to avoid complex `isModLoaded` call.